Swift/Tests/ReplicatorTest+Collection.swift

```diff
@@ -163,7 +162,7 @@ class ReplicatorTest_Collection: ReplicatorTest {
         XCTAssertNotNil(colConfig.pullFilter)
     }
     
-    func testCreateConfigWithEndpointOnly() {
+    func _testCreateConfigWithEndpointOnly() {
```

Comment:
Why disabled?

Reply:
will remove, since the test validates the fatalError, which is not possible.